src/components/ResponsiveAppBar.jsx

```diff
@@ -113,6 +125,10 @@ const ResponsiveAppBar = () => {
                   </ListSubheader>
                 )
               )}
+              <Typography textAlign="center">
+                {t("Tour")}
+                <Welcome />
+              </Typography>
```

Comment:
The `?` icon added seems a bit strange.

What would be great is to redo the `Welcome` to move the logic `isTourOpen` outside of it (and put it into the toolbar) such that you can control it's open/close state as follow

```suggestion
              <MenuItem component="button" onClick={() => {openTheWelcomeTour}}>
                {t("menu.tour")}
              </MenuItem>
```